frontend/src/components/home/About.js

```diff
@@ -1,7 +1,26 @@
 import React from 'react'
 
 function About() {
-    return <div>-about component placeholder for home page-</div>
```

Comment:
needs to exclude all the changes in components/home because I'm pretty sure this is from Aagaman's commit and I don't want to include this on the PR